data/pathfinder/paizo/roleplaying_game/bestiary_3/b3_abilities_race_oa.lst

```diff
@@ -0,0 +1,8 @@
+
+# Support for OA archetype familiar
+Race Traits ~ Kami (Shikigami)			CATEGORY:Internal																																																																																																																																									ABILITY:Special Ability|AUTOMATIC|TYPE=ShikigamiRacialAbility																																																																																																																																																																																																																																																					BONUS:VAR|TelepathyRange|200	BONUS:VAR|FastHealingRate|2
+
+# Shikigami 163
+Improvised Weapon Mastery		KEY:Shikigami ~ Improvised Weapon Mastery		CATEGORY:Special Ability	TYPE:ShikigamiRacialAbility.SpecialQuality.Extraordinary																																																																																																																																																																																																																																					DESC:A shikigami gains Catch Off-Guard and Throw Anything as bonus feats, and adds its Charisma modifier instead of its Strength modifier to damage done with any improvised weapon, as attacks it makes with such weapons seem supernaturally lucky in landing damaging blows. Although a shikigami is Tiny, it never provokes attacks of opportunity when it attacks an adjacent foe with a melee weapon. If a shikigami critically hits an opponent with an improvised weapon, it deals ?3 damage.																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																				SOURCEPAGE:p.163
+Spell-Like Abilities			KEY:Shikigami ~ Spell-Like Abilities		CATEGORY:Special Ability	TYPE:ShikigamiRacialAbility.SpecialQuality.SpellLike																																																																																																																																																																																																																																						DESC:(CL 6th; concentration +8) At will-invisibility (self only), statue (self only) 3/day-hide from animals, purify food and drink 1/week-commune with nature (CL 12th)
```

Comment:
This style of set up goes in the OA folder and is loaded if the B3 version is not loaded. We avoid messing with the originals and do the modifications at the source. OR, you can merely have OA load the creature and avoid this work around.